src/main/java/org/ballerinalang/docgen/docs/html/HtmlDocumentWriter.java

```diff
@@ -91,14 +96,33 @@ public void write(Collection<BLangPackage> packages) throws IOException {
         if (BallerinaDocUtils.isDebugEnabled()) {
             out.println("Generating HTML API documentation...");
         }
-        Files.createDirectories(Paths.get(outputFilePath));
```

Comment:
In fact, I see https://github.com/ballerinalang/docerina/pull/70/files has removed the same line. Hence, the question.